libs/application/ui-fields/src/lib/FileUploadFormField/FileUploadFormField.tsx

```diff
@@ -23,7 +29,32 @@ export const FileUploadFormField = ({ application, field, error }: Props) => {
     forImageUpload,
   } = field
 
+  const setNestedProperty = (obj: any, path: string, value: any): any => {
+    const keys = path.split('.')
+    const lastKey = keys.pop()
+    if (!lastKey) return obj
+    const nested = keys.reduce((acc, key) => (acc[key] = acc[key] || {}), {
+      ...obj,
+    })
+    nested[lastKey] = value
+    return obj
+  }
```

Comment:
This could be useful in the future, let's move it to shared lib`libs/application/core/src/lib/formUtils.ts`

Reply:
Thank god! Fixed